msteams-platform/index.yml

```
@@ -14,100 +15,120 @@ metadata:
+        - url: /microsoftteams/platform/tutorials/get-started-dotnet-app-studio
+          itemType: how-to-guide
+          text: Build a Teams app using C#/.NET and App Studio
+        - url: /microsoftteams/platform/samples/teams-low-code-solutions
+          itemType: how-to-guide
+          text: Build low-code apps, bots, and flows with Microsoft Power Platform
+        - url:  /microsoftteams/platform/resources/schema/manifest-schema
+          itemType: reference
+          text: Manifest schema for Microsoft Teams
       
     # Card
     - title: Learn
       links:
         - url: /learn/modules/intro-microsoft-teams-apps
           itemType: learn
-          text: Build apps for Microsoft Teams — introduction
+          text: Build apps for Microsoft Teams
         - url: /learn/modules/embedded-web-experiences
           itemType: learn
-          text: Tabs — create embedded web experiences 
+          text: Create embedded web experiences with tabs
         - url:  /learn/modules/msteams-conversation-bots/
           itemType: learn
-          text: Bots — create conversational interactions
+          text: Create conversational interactions with bots
         - url: /learn/modules/msteams-messaging-extensions/
           itemType: learn
-          text: Messaging extensions — create task-oriented interactions 
+          text: Create task-oriented interactions with messaging extensions
         - url: /learn/modules/msteams-task-modules/
           itemType: learn
-          text: Task modules — collect input and gather information
+          text: Collect input and gather information with task modules
         - url: /learn/modules/msteams-webhooks-connectors/
           itemType: learn
-          text: Webhooks and Office 365 connectors — connect web services to channels and teams
+          text: Connect web services to channels and teams with webhooks and Office 365 connectors
     # Card
-    - title: Develop
-      links:
-        - url: /microsoftteams/platform/tutorials/get-started-nodejs-app-studio
-          itemType: get-started
-          text: Build a Teams app using JavaScript/Node.js and App Studio
-        - url: /microsoftteams/platform/tutorials/get-started-dotnet-app-studio
-          itemType: get-started
-          text: Build a Teams app using C#/.NET and App Studio
-     # Card
-    - title: Deploy
+    - title: Integrate
       links:
+        - url:  /microsoftteams/platform/samples/integrating-web-apps
+          itemType: reference
+          text: Integrate web apps with Microsoft Teams
+        - url: /microsoftteams/platform/bots/how-to/add-power-virtual-agents-bot-to-teams
+          itemType: reference
+          text: Integrate your Power Virtual Agents bot with Microsoft Teams
+        - url: /microsoftteams/platform/tabs/how-to/tabs-in-sharepoint
+          itemType: reference
+          text: Integrate Microsoft Teams and SharePoint 
```

```suggestion
          text: Add a Teams tab to SharePoint
```
More closely matches doc.

---

@heath-hamilton, Was trying to maintain parallelism.

---

Makes sense, @surbhigupta12. Please see my updated suggestion. I think the key is just pointing out this is only supported with tabs.